src/front/app/components/ChallengeTopics/index.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { getChallengeTopics, toggleChallengeTopic } from './../../actions/challengeTopics';
+import ChallengeTopic from './ChallengeTopic';
+
+class ChallengeTopicList extends Component {
+    constructor(props) {
+        super(props);
+        this.handleClick = this.handleClick.bind(this);
```

Comment:
And it is true, but
I suggest that binding all prototype methods on instance is bad practice
you can avoid this using ES6 arrows
```js
render() {
  return (
    <button onClick={() => this.onClick()}>
        Click me
   </button>
  );
}
```